go/client/commands_common.go

```diff
@@ -48,6 +48,7 @@ func GetCommands(cl *libcmdline.CommandLine, g *libkb.GlobalContext) []cli.Comma
 		NewCmdPing(cl, g),
 		NewCmdProve(cl, g),
 		NewCmdRekey(cl, g),
+		NewCmdRIIT(cl, g),
```

Comment:
ok to have this in production?

Reply:
yeah as long as it's hidden.  i want it on production for debugging